src/test/ci/song_selector.test.ts

```diff
@@ -562,24 +547,20 @@ describe("getFilteredSongList", () => {
     });
 });
 
-describe("selectRandomSong", () => {
+describe("selectRandomSong", function () {
     describe("gender override", () => {
         beforeEach(async () => {
             await guildPreference.setGender([Gender.ALTERNATING]);
         });
 
         describe("override to female", () => {
-            it("should match the expected song count of female + coed songs", async () => {
+            it("should only return female/coed songs", async () => {
                 const femaleOrCoedSongs = [];
                 const filteredSongs = (
                     await SongSelector.getFilteredSongList(guildPreference)
                 ).songs;
 
-                const femaleOrCoedSongCount = [...filteredSongs].filter((x) =>
-                    [Gender.FEMALE, Gender.COED].includes(x.members)
-                ).length;
-
-                for (let i = 0; i < femaleOrCoedSongCount; i++) {
+                for (let i = 0; i < 10; i++) {
```

Comment:
Can the for loop go over all female or coed songs?

Reply:
See below